Dotnet.Homeworks.Tests.RunLogic/Utils/Docker/DockerCompose.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace Dotnet.Homeworks.Tests.RunLogic.Utils.Docker;
+
+public class DockerCompose
+{
+    public ServicesSection? Services { get; set; }
+}
+
+public class ServicesSection
+{
+    public DotnetPostgres? DotnetPostgres { get; set; }
```

Comment:
Нужно пробежаться по всем файлам и там, где нет инициализации или установка новых значений убрать аццессор set;. Если какое-то значение устанавливается через pattern-matching (данные после анализа файла, данные из appsettings.json и т.п.), то  такие места не надо исправлять
```suggestion
    public DotnetPostgres? DotnetPostgres { get; }
```

Reply:
это итак сделано везде. в указанном файле сеттеры нужны для десериализации